src/erlcloud_ddb2.erl

```diff
@@ -2825,3 +2829,425 @@ to_binary(X) when is_list(X) ->
     list_to_binary(X);
 to_binary(X) when is_integer(X) ->
     integer_to_binary(X).
+
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% DynamoDB API:
+%% [https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ListBackups.html]
+%%
+%% ===Example===
+%%
+%% Get the table backups.
+%% `
+%% {ok, Tables} =
+%%     erlcloud_ddb2:list_backups(),
+%% '
+%% Get the last 4 table backups for "Forum" table between April 5, 2018 and April 6, 2018.
+%% `
+%% {ok, Tables} =
+%%     erlcloud_ddb2:list_backups(
+%%       [{limit, 4},
+%%        {table_name, <<"Forum">>},
+%%        {time_range_lower_bound,1522926603.688},
+%%        {time_range_upper_bound,1523022454.098}]),
+%% '
+%% @end
+%%------------------------------------------------------------------------------
+-type list_backups_opt() :: {limit, pos_integer() | undefined} |
+{exclusive_start_backup_arn, undefined } | {table_name, table_name() | undefined} |
```

Comment:
This looks wrong. should it be `{exclusive_start_backup_arn, binary() | undefined }`?